common/app/routes/Challenges/Tool-Panel.jsx

```diff
@@ -125,9 +125,8 @@ export default class ToolPanel extends PureComponent {
           <Button
             bsSize='large'
             bsStyle='primary'
-            componentClass='a'
-            href={ `https://gitter.im/freecodecamp/${helpChatRoom}` }
-            target='_blank'
+            componentClass='label'
```

Comment:
for accessibility reasons, tags that have click events should be either an anchor tag or a button. This makes it possible to `tab` to the element in question.

Reply:
I'm having some trouble replacing `label` with `button`. 

![Screenshot](https://i.imgur.com/e5I3fK4.png)

I'm not sure which CSS property is responsible for this behavior.

I need help with this and I would appreciate any advice.

Reply:
To get started quickly here's an example page:

https://beta.freecodecamp.org/en/challenges/es6/compare-scopes-of-the-var-and-let-keywords

You can observe similar problem after replacing `label` for Bug button with a button tag:

![Screenshot](https://i.imgur.com/8goAm6T.png)

Reply:
hmm, I forgot how tricky the css is for button groups. For expediency we will keep the PR as is with the label.

Reply:
in `challenge.less`

```less
.tool-panel-btn-grp {
  display: flex;

  .btn {
  flex-grow: 1;
  }
}
```

In Tool-Panel.jsx
```jsx
<ButtonGroup
  className='input-group tool-panel-btn-grp'
  justified={ true }
  >
  <Button
    bsSize='large'
    bsStyle='primary'
    onClick={ this.makeReset }
    >
    Reset
  </Button>
  <Button
    bsSize='large'
    bsStyle='primary'
    onClick={ openHelpModal }
    >
    Help
  </Button>
  <Button
    bsSize='large'
    bsStyle='primary'
    onClick={ openBugModal }
    >
    Bug
  </Button>
</ButtonGroup>
```

And we have the buttons and UI we would like :+1: